rwgame/RWGame.cpp

```diff
@@ -457,7 +457,7 @@ void RWGame::tick(float dt)
 		}
 		
 		// Clean up old VisualFX
-		for( int i = 0; i < world->effects.size(); ++i )
+		for( ssize_t i = 0; i < static_cast<ssize_t>(world->effects.size()); ++i )
```

Comment:
This block isn't great, but it can be cleaned up with a more thorough pass on rwgame.